pubsub/azure/eventhubs/metadata.yaml

```
@@ -0,0 +1,74 @@
+# yaml-language-server: $schema=../../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: azure.eventhubs
+version: v1
+status: stable
+title: "Azure Event Hubs"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-eventhubs/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "Authenticate using a connection string."
+    metadata:
+      - name: connectionString
+        required: true
+        sensitive: true
+        description: "Connection string for the Event Hub or the Event Hub namespace. Mutally exclusive with eventHubNamespace field."
+        example: '"Endpoint=sb://{EventHubNamespace}.servicebus.windows.net/;SharedAccessKeyName={PolicyName};SharedAccessKey={Key};EntityPath={EventHub}"
+        or "Endpoint=sb://{EventHubNamespace}.servicebus.windows.net/;SharedAccessKeyName={PolicyName};SharedAccessKey={Key}"'
+builtinAuthenticationProfiles:
+  - name: "azuread"
+    metadata:
+      - name: eventHubNamespace
+        description: "The Event Hub Namespace name. Mutually exclusive with connectionString field."
```

`connectionString` is in another auth profile so this is implicit. 

```suggestion
        description: "The Event Hub Namespace name. "
```

---

Addressed